### ISSUE TITLE
Onchange doesn't trigger when dropdown.js used with React components

### ISSUE THREAD

**Issue by YPCrumble** [FezVrasta/dropdown.js#38]
See [this fiddle](https://jsfiddle.net/ypcrumble/obd6kz0y/).

If you comment out lines 7-8, the onChange event fires fine. When you include these it no longer fires. I'm not exactly sure how to solve, but if you lead me in the right direction I'd be happy to attempt a pull request.

**Comment by jonigl** [comment]
I have this problem too 😕 !

@YPCrumble could you solved this?

Thanks!
Jonathan

**Comment by YPCrumble** [comment]
@jonigl for this and other reasons I went back to Bootstrap...there's a lot of DOM manipulation in Material Design that doesn't lend itself to javascript frontend apps.

**Comment by jonigl** [comment]
@YPCrumble yes, I think so. This kind of themes don't get along with React...

**Comment by FezVrasta** [comment]
This plugin is not thought to be used with react, sorry
